js/compat/customize-snapshots.js

```diff
@@ -0,0 +1,465 @@
+/* global jQuery, wp */
+/* eslint consistent-this: ["error", "snapshot"] */
+
+( function( api, $ ) {
+	'use strict';
+
+	if ( ! api.Snapshots ) {
```

Comment:
I'd suggest removing this conditional. If this is not defined then it is clearly an error and better to not fail silently.